misc-scripts/xref_mapping/XrefMapper/DisplayXrefs.pm

```diff
@@ -446,15 +446,15 @@ sub set_display_xrefs{
 
   print "Using xref_off set of $xref_offset\n" if($self->verbose);
 
-  my $reset_sth = $core_dbi->prepare("UPDATE gene SET display_xref_id = null");
-  $reset_sth->execute();
-  $reset_sth->finish;
+  #my $reset_sth = $core_dbi->prepare("UPDATE gene SET display_xref_id = null");
```

Comment:
Please remove these, no need to keep them around.

Reply:
You really dislike commented out code, don't you :D To be honest I do too, I just forget to delete before committing. Will remove! Thanks